src/main/java/org/venice/beachfront/bfapi/model/Detection.java

```diff
@@ -3,31 +3,66 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
 import javax.persistence.Table;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.vividsolutions.jts.geom.Geometry;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
 
 @Entity
+@IdClass(DetectionPK.class)
```

Comment:
I'm going to change this to EmbeddedId... It looks like Spring tends to like that implementation a little better.